components/tabs/demo/tabs.html

```diff
@@ -34,6 +34,40 @@ <h2>Tabs</h2>
 				</template>
 			</d2l-demo-snippet>
 
+			<h2>Tabs (paired d2l-tab with d2l-panel)</h2>
+
+			<d2l-demo-snippet>
+				<div>This format is not yet functional. Please do not use yet.</div>
+				<template>
+					<d2l-tabs>
+						<d2l-tab id="all" text="All" slot="tabs"></d2l-tab>
+						<d2l-tab id="biology" text="Biology" slot="tabs"></d2l-tab>
```

Comment:
This demo shows the new style where each `d2l-tab` is paired with a `d2l-tab-panel` via `id` and `labelled-by`. They are all put into either the `tabs` or `panels` slots.